build/assets/themes/dashboard/js/main.js

```diff
@@ -54,10 +54,10 @@ var setupFavorites = function() {
                 success: function(r) {
                     if (r.action == 'remove') {
                         $link.attr('data-bookmark-action', 'add-favorite');
-                        $link.html('<i class="fa fa-lg fa-bookmark-o"></i>');
+                        $link.find('.icon-bookmark').removeClass('bookmarked');
```

Comment:
What's the point of this? Why not just remove it from the HTML?

Reply:
This adding and removing of the class happens via javascript so page wont refresh and will apply the visual icon status change.

Reply:
I see, this is being toggled